app/main/server/app/ui/main-window.js

```diff
@@ -160,6 +188,56 @@ module.exports = class MainWindow {
     this.rpc.call('notifyPluginsReloading');
   }
 
+  applyTheme(themeObj) {
+    if (this.browserWindow === null) {
+      return;
+    }
+
+    this.browserWindow.setSize(
+      themeObj.themeObj.window.width,
+      themeObj.themeObj.window.height
+    );
+
+    // set background color?
+    const windowColor = ThemeObject.determineTransparentColor(
+      themeObj.themeObj,
+      themeObj.themeObj.window.color
+    );
+
+    if (windowColor) {
+      this.browserWindow.webContents.insertCSS(
```

Comment:
Setting a background color cause the loss of vibrancy theme in macOS 😢 
So it would be better to make a setting for the vibrancy only for macOS